### ISSUE TITLE
Could we get more frames?

### ISSUE THREAD

**Issue by MaciekVoxel** [ephtracy/ephtracy.github.io#238]
120 frames is a lot but still it would be nice if we had more. Maybe something you could change manually in the config file?

**Comment by DimasVoxel** [comment]
I see it's a fresh issue, but I agree with the author.
120 are enough for a tiny animation, but it limits anything that tries to be more advanced. 
I have done 6 Animations so far, and 3 of those exceeded the 120 frame limit so much I had to create two new files just to continue the animation. 
I hope there will be some kind of solution

**Comment by SutaruBoxu** [comment]
Definitely! If we could even just get double the current frames in a not-too-distant update, I believe it would be very helpful.
